src/plots/barPlot.ts

```diff
@@ -662,11 +740,16 @@ export class Bar<X, Y> extends XYPlot<X, Y> {
       return Math.abs(scaledBaseline - originalPositionFn(d, i, dataset));
     };
 
-    attrToProjector["width"] = this._isVertical ? widthF : heightF;
-    attrToProjector["height"] = this._isVertical ? heightF : widthF;
+    const gapF = attrToProjector["gap"];
```

Comment:
we should docs this `gap` thing or otherwise have proper first-class support for it